src/conf.py

```diff
@@ -96,17 +140,11 @@
     ],
 }
 
-# Logo and favicon to display.
-html_logo = "graphics/png/orig/cylc-logo.png"
-# sphinx specifies this should be .ico format
-html_favicon = "graphics/cylc-favicon.ico"
+html_static_path = ['_static']
 
 # Disable timestamp otherwise inserted at bottom of every page.
 html_last_updated_fmt = ''
 
-# Remove "Created using Sphinx" text in footer.
```

Comment:
I think that information could be useful to us as well? Some times the documentation needs amending even after a release. So a person updating it can check the footer and confirm the date changed, and users can also quickly see when it was updated.

Reply:
I think that would be a rare case, & the developer could just check the specific changes that were made to confirm the same. And then the user, who could also see an update, would also pick up on an apparent lack of update too; for example because of the Cylc 8 circumstances our ``7.8.3`` docs would have that timestamp changing very infrequently such that they could think we barely maintain our docs. Also the extra text it requires makes the footer line take up two lines on a standard-sized screen window which takes up too much space for a footer IMO.

So for those reasons, I personally don't think it is best to include it, but since we disagree I suppose we can let @oliver-sanders decide? Ultimately, it doesn't matter much, so if we include it that's still fine by me 😄

Reply:
I think this is pretty irrelevant!

Don't know that you can turn it off in the RTD theme anyway.